src/engine/helper.ts

```diff
@@ -0,0 +1,87 @@
+import Square from "./square";
+import GameSettings from "./gameSettings";
+import Board from "./board";
+
+export function moveInBounds(newMove: Square) {
+    if (
+        (newMove.col > GameSettings.BOARD_SIZE-1) ||
+        (newMove.col < 0)||
+        (newMove.row > GameSettings.BOARD_SIZE-1) ||
+        (newMove.row < 0)
+    ){
+        return false;
+    } 
+    return true;
+}
+
+export function getDiagonalMoves(board:Board, currentPosition: Square){

```

Comment:
```suggestion
export function getDiagonalMoves(board:Board, currentPosition: Square): Square[] {
```

we want to be adding a return type to all our functions that return a value